third_party/move/move-vm/runtime/src/runtime_ref_checks.rs

```diff
@@ -469,6 +472,15 @@ impl RuntimeRefCheck for FullRuntimeRefCheck {
             MoveLoc(index) => {
                 ref_state.move_loc(*index)?;
             },
+            DropLoc(index) => {
+                // MoveLoc
```

Comment:
You can ignore ref checks for now: they never run for experiments (unless you want to add them)